docs/self-hosted/docker-guide.md

```diff
@@ -157,10 +144,9 @@ docker-compose pull
 docker-compose up -d
```

Comment:
```suggestion
docker compose up -d
```

Reply:
Oh, annoying. I didn't know that and in the video it's with a dash. Is there a material difference? If not I may suggest leacing this for consistency

Reply:
Also in documentary context, I prefer to use the long, written out version of CLI flags whenever available so that it is easier to see exactly what it does

```suggestion
docker compose up --down
```

Reply:
@paescuj the `-d` there stands for `--detach` (not `--down`) ;)

Reply:
And, actually, I'm going to remove it. It's much more explicit to have it running in the terminal.

Reply:
> @paescuj the `-d` there stands for `--detach` (not `--down`) ;)

Haha right 🤣

Reply:
I dont think i agree with removing them from the whole quickstart as it is the best practice way of running dockers. Allowing for dockers to be run in the background preventing you from accidentally hard-shutting down the entire service and allowing you to get the logs for only Directus instead of getting bombarded with all database, cache and directus logs mixed together.

Reply:
> I dont think i agree with removing them from the whole quickstart as it is the best practice way of running dockers. Allowing for dockers to be run in the background preventing you from accidentally hard-shutting down the entire service and allowing you to get the logs for only Directus instead of getting bombarded with all database, cache and directus logs mixed together.

Hmm right, I think I agree with that 👍 
Maybe we could supplement that with the following command enabling users to see the generated password:
```bash
docker compose logs --follow
```

Reply:
I don't agree and think we should remove them. This is where we venture into teaching more than is required to use Directus, and it's a slippery slope.

Reply:
tricky tricky, because whether we want to or not if someone has not used Docker before then we are teaching them here and now and i would hate to get them started on non-best practices and then having to explain best practices over and over in the discord 🤔

Reply:
I don't think that's our place, but it's not a hill I'll die on. Shall we get the input of @rijkvanzanten on this?